ui/components/app/gas-customization/gas-slider/index.scss

```diff
@@ -17,7 +17,7 @@
     height: 34px;
     width: 34px;
     background-color: var(--color-primary-default);
-    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.08);
+    box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
     border-radius: 50%;
     position: relative;
     z-index: 10;
```

Comment:
Not sure this component is used and updating the shadow did nothing anyway. Going to remove

<img width="681" alt="gas-slider-no-shadow" src="https://user-images.githubusercontent.com/8112138/179421171-eb1ee663-e08e-4a18-af61-f4847efae929.png">